core/services/relay/evm/evm.go

```diff
@@ -281,7 +280,6 @@ func (r *Relayer) NewPluginProvider(rargs commontypes.RelayArgs, pargs commontyp
 	}
 
 	return NewPluginProvider(
-		r.chainReader,
```

Comment:
@ilija42 this is what you mean? [`0e9a5b0` (#14146)](https://github.com/smartcontractkit/chainlink/pull/14146/commits/0e9a5b09034d03d472bf5cd22889ec50334709a5) 

Do I need to bind a contract in the initialization?

Reply:
> @ilija42 this is what you mean? [0e9a5b0 (#14146)](https://github.com/smartcontractkit/chainlink/pull/14146/commits/0e9a5b09034d03d472bf5cd22889ec50334709a5)
Yes, looks good!

> Do I need to bind a contract in the initialization?
No, I just wanted to avoid unexpected confusion and unexpected behaviour here

Reply:
tagging @DeividasK since he worked on PluginProvider

Reply:
Message from @DeividasK 
> :wave: hey, we only use relayer.NewContractReader in core/services/relay/evm/write_target.go, so it should be ok to remove the chainReader field.